docs/README.md

```diff
@@ -0,0 +1,186 @@
+
+An effort was made to reduce the impact that sphinx would add due to the configuration and additional files that are added as part of the build process. Not to mention that sphinx out of the box only support `reStructuredText`.  This implementation adds support for `markdown` to contribute to the documentation.  The addition of sphinx puts us in-line with the rest of the RPC organization and it gives us the ability to generate internal and external documentation from the same repository; with the addition of a simple environment variable at build time.
+
+All external documentation is in directories that are postfixed `-external` and anything that is internal is postfixed with `-internal`.
+
+### Adding a new internal section
+If you wanted to add a new section that is meant to be for internal use only or want to keep it internal for sometime create a new directory and postfix `-internal` and update the `conf.py` file and add your directory to the list `exclude_patterns.extend([])`
+
+## Building the documentation
+To build the documentation is as simple as running the following command. The execution  will result in the creation of a directory named `_build`.
+
+### Build internal and external documentation
+From with in the `docs` directory execute the following command:
+```
+make docs
+```
+
+### Only build external documentation
+The following command will only build customer/public facing documentation.
+```
+make docs-external
+```
+
+### View newly built documentation
+If you are on `osx` you can view the documentation by executing the following:
+
+from with in the `rpc-ceph/docs directory`
+```
+open _build/html/index.html
+```
+
+from with in the `rpc-ceph directory`
+```
+open rpc-ceph/_build/html/index.html
+```
+
+## Where did the old documentation go?
+All the previous documentation is still available as it was simply moved to a different directory. The following maps the new to the old.
+
+| Original Location | New Location |
+| :--- | :--- |
+| rgw_setup.md | ops-internall/rgw_setup.md |
+| rpc-ceph-install.md | ops-internal/rpc-ceph-install.md |
+| deployment-scenariosi/ | ops-internal |
+| deployment-scenarios/CephNodeAddition.txt | ops-internal/ceph-node-addition.md |
+| deployment-scenarios/CephNodeAdditionJewel.txt | ops-internal/ceph-node-addition-Jewel.md |
+| deployment-scenarios/Ceph_Standalone_Install.md | ops-internal/ceph-standalone-install.md |
+| deployment-scenarios/RollingUpdates.yml | ops-internal/rollingUpdates.md |
+
+# Sphinx Docker Image
+
+## Building sphinx
+Build the Sphinx Docker image:
+```
+docker build -t quay.io/rackspace/sphinx .
+```
+
+## Push the newly created sphinx to quay.io
+Assuming you have access to the `quay.io/rackspace` repository you can push the image by executing the following:
+```
+docker push quay.io/rackspace/sphinx
+```
+
+# Unsure about this section 
```

Comment:
I think we should drop this section until we know what content it should have and not carry forward the kubernetes content.  Or update it.